pom.xml

```diff
@@ -167,6 +173,28 @@
                 <spark.version>${spark311.version}</spark.version>
                 <spark.test.version>${spark311.version}</spark.test.version>
             </properties>
+            <modules>
+                <module>api_validation</module>
```

Comment:
We're not building versions of these for Databricks?

Reply:
I removed them because we don't validate our api vs databricks version so why build it?